docs/index.md

```diff
@@ -321,49 +355,47 @@ npm install -g flow-language-server
 
 3. Add to LSP settings' clients:
 
-        "fortls": {
-          "command": ["fortls"],
-          "enabled": true,
-          "languageId": "fortran",
-          "scopes": [
-            "source.modern-fortran",
-            "source.fixedform-fortran"
-          ],
-          "syntaxes": [
-            "Packages/Fortran/grammars/FortranModern.sublime-syntax",
-            "Packages/Fortran/grammars/FortranFixedForm.sublime-syntax"
-          ]
-        }
+```json
```

Comment:
Shouldn't this be `js`?

Reply:
I was using `js` for all code blocks which contain comments (because Mkdocs renders comments as invalid for JSON), and `json` for all code blocks without comments (because in that case the different highlighting colors for keys and values look a bit nicer in my opinion and `json` was used in the majority of cases beforehand). In this case here, there is no comment, so `js` wouldn't be necessary.
I could change all code blocks in the file to `js` for consistency, if you want though.

Reply:
Well, I'm okay with either.